boxnav/boxnav/boxnavigator.py

```diff
@@ -274,7 +260,7 @@ def __init__(
         self.num_resets = 0
         self.trial_num = 0
 
-        self.previous_action = Action.NO_ACTION
+        self.image_delay = 1
```

Comment:
```suggestion
        # Initial delay is 1s so that UE can get warmed up
        self.image_delay = 1
```

Reply:
I had forgotten to commit this initialization last time but it didn't actually work for me with an initial delay of 1s. It did work with an initial delay of 3s. I will include this in my next commit but wanted to make sure you knew.